[01/Oct/2021:05:55:00] ENGINE Bus STARTING
[01/Oct/2021:05:55:00] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:55:00] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 1000): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:55:00] ENGINE Bus STOPPING
[01/Oct/2021:05:55:00] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 1000)) already shut down
[01/Oct/2021:05:55:00] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:55:00] ENGINE Bus STOPPED
[01/Oct/2021:05:55:00] ENGINE Bus EXITING
[01/Oct/2021:05:55:00] ENGINE Bus EXITED
[01/Oct/2021:05:55:17] ENGINE Bus STARTING
[01/Oct/2021:05:55:17] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:55:17] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 1100): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:55:17] ENGINE Bus STOPPING
[01/Oct/2021:05:55:17] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 1100)) already shut down
[01/Oct/2021:05:55:17] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:55:17] ENGINE Bus STOPPED
[01/Oct/2021:05:55:17] ENGINE Bus EXITING
[01/Oct/2021:05:55:17] ENGINE Bus EXITED
[01/Oct/2021:05:55:19] ENGINE Bus STARTING
[01/Oct/2021:05:55:19] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:55:19] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 1100): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:55:19] ENGINE Bus STOPPING
[01/Oct/2021:05:55:19] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 1100)) already shut down
[01/Oct/2021:05:55:19] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:55:19] ENGINE Bus STOPPED
[01/Oct/2021:05:55:19] ENGINE Bus EXITING
[01/Oct/2021:05:55:19] ENGINE Bus EXITED
[01/Oct/2021:05:55:27] ENGINE Bus STARTING
[01/Oct/2021:05:55:27] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:55:27] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 1111): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:55:27] ENGINE Bus STOPPING
[01/Oct/2021:05:55:27] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 1111)) already shut down
[01/Oct/2021:05:55:27] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:55:27] ENGINE Bus STOPPED
[01/Oct/2021:05:55:27] ENGINE Bus EXITING
[01/Oct/2021:05:55:27] ENGINE Bus EXITED
[01/Oct/2021:05:56:10] ENGINE Bus STARTING
[01/Oct/2021:05:56:10] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:10] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 2222): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:10] ENGINE Bus STOPPING
[01/Oct/2021:05:56:10] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 2222)) already shut down
[01/Oct/2021:05:56:10] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:10] ENGINE Bus STOPPED
[01/Oct/2021:05:56:10] ENGINE Bus EXITING
[01/Oct/2021:05:56:10] ENGINE Bus EXITED
[01/Oct/2021:05:56:16] ENGINE Bus STARTING
[01/Oct/2021:05:56:16] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:16] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 2222): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:16] ENGINE Bus STOPPING
[01/Oct/2021:05:56:16] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 2222)) already shut down
[01/Oct/2021:05:56:16] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:16] ENGINE Bus STOPPED
[01/Oct/2021:05:56:16] ENGINE Bus EXITING
[01/Oct/2021:05:56:16] ENGINE Bus EXITED
[01/Oct/2021:05:56:25] ENGINE Bus STARTING
[01/Oct/2021:05:56:25] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:26] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 2333): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:26] ENGINE Bus STOPPING
[01/Oct/2021:05:56:26] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 2333)) already shut down
[01/Oct/2021:05:56:26] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:26] ENGINE Bus STOPPED
[01/Oct/2021:05:56:26] ENGINE Bus EXITING
[01/Oct/2021:05:56:26] ENGINE Bus EXITED
[01/Oct/2021:05:56:36] ENGINE Bus STARTING
[01/Oct/2021:05:56:36] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:36] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 6555): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:36] ENGINE Bus STOPPING
[01/Oct/2021:05:56:36] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 6555)) already shut down
[01/Oct/2021:05:56:36] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:36] ENGINE Bus STOPPED
[01/Oct/2021:05:56:36] ENGINE Bus EXITING
[01/Oct/2021:05:56:36] ENGINE Bus EXITED
[01/Oct/2021:05:56:43] ENGINE Bus STARTING
[01/Oct/2021:05:56:43] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:43] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 6555): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:43] ENGINE Bus STOPPING
[01/Oct/2021:05:56:43] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 6555)) already shut down
[01/Oct/2021:05:56:43] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:43] ENGINE Bus STOPPED
[01/Oct/2021:05:56:43] ENGINE Bus EXITING
[01/Oct/2021:05:56:43] ENGINE Bus EXITED
[01/Oct/2021:05:56:53] ENGINE Bus STARTING
[01/Oct/2021:05:56:53] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:56:54] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 0): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:56:54] ENGINE Bus STOPPING
[01/Oct/2021:05:56:54] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 0)) already shut down
[01/Oct/2021:05:56:54] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:56:54] ENGINE Bus STOPPED
[01/Oct/2021:05:56:54] ENGINE Bus EXITING
[01/Oct/2021:05:56:54] ENGINE Bus EXITED
[01/Oct/2021:05:57:03] ENGINE Bus STARTING
[01/Oct/2021:05:57:03] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:57:03] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 10): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:57:03] ENGINE Bus STOPPING
[01/Oct/2021:05:57:03] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 10)) already shut down
[01/Oct/2021:05:57:03] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:57:03] ENGINE Bus STOPPED
[01/Oct/2021:05:57:03] ENGINE Bus EXITING
[01/Oct/2021:05:57:03] ENGINE Bus EXITED
[01/Oct/2021:05:57:17] ENGINE Bus STARTING
[01/Oct/2021:05:57:17] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:57:17] ENGINE Error in HTTP server: shutting down
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\process\servers.py", line 225, in _start_http_thread
    self.httpserver.start()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1836, in start
    self.prepare()
  File "C:\Python38\lib\site-packages\cheroot\server.py", line 1791, in prepare
    raise socket.error(msg)
OSError: No socket could be created -- (('127.0.0.0', 7000): [WinError 10049] The requested address is not valid in its context)

[01/Oct/2021:05:57:17] ENGINE Bus STOPPING
[01/Oct/2021:05:57:17] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.0', 7000)) already shut down
[01/Oct/2021:05:57:17] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:57:17] ENGINE Bus STOPPED
[01/Oct/2021:05:57:17] ENGINE Bus EXITING
[01/Oct/2021:05:57:17] ENGINE Bus EXITED
[01/Oct/2021:05:57:27] ENGINE Bus STARTING
[01/Oct/2021:05:57:27] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:57:27] ENGINE Serving on http://127.1.1.1:7000
[01/Oct/2021:05:57:27] ENGINE Bus STARTED
[01/Oct/2021:05:57:32] ENGINE Restarting because C:\src\html\SEGCloud\index.py changed.
[01/Oct/2021:05:57:32] ENGINE Stopped thread 'Autoreloader'.
[01/Oct/2021:05:57:32] ENGINE Bus STOPPING
[01/Oct/2021:05:57:32] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.1.1.1', 7000)) shut down
[01/Oct/2021:05:57:32] ENGINE Bus STOPPED
[01/Oct/2021:05:57:32] ENGINE Bus EXITING
[01/Oct/2021:05:57:32] ENGINE Bus EXITED
[01/Oct/2021:05:57:32] ENGINE Waiting for child threads to terminate...
[01/Oct/2021:05:57:32] ENGINE Re-spawning C:\Python38\python.exe C:\src\html\SEGCloud\index.py
[01/Oct/2021:05:57:34] ENGINE Bus STARTING
[01/Oct/2021:05:57:34] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:05:57:34] ENGINE Serving on http://127.1.1.1:1000
[01/Oct/2021:05:57:34] ENGINE Bus STARTED
[01/Oct/2021:05:59:01] HTTP 
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
  File "C:\Python38\lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "C:\Python38\lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "C:\src\html\SEGCloud\index.py", line 97, in home
    byt = getFolderSize(MYID)
  File "C:\src\html\SEGCloud\index.py", line 13, in getFolderSize
    total_size = os.path.getsize(folder)
  File "C:\Python38\lib\genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [WinError 2] The system cannot find the file specified: '0\\'
[01/Oct/2021:05:59:01] HTTP 
Request Headers:
  Remote-Addr: 127.0.0.1
  HOST: zuuvs3xsp57e42a5c5lqoby5sli5hxfvbbezfgnfsqopjwqlribyggad.onion
  USER-AGENT: Mozilla/5.0 (Windows NT 10.0; rv:78.0) Gecko/20100101 Firefox/78.0
  ACCEPT: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
  ACCEPT-LANGUAGE: en-US,en;q=0.5
  ACCEPT-ENCODING: gzip, deflate
  CONNECTION: keep-alive
  UPGRADE-INSECURE-REQUESTS: 1
[01/Oct/2021:05:59:08] HTTP 
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
  File "C:\Python38\lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "C:\Python38\lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "C:\src\html\SEGCloud\index.py", line 97, in home
    byt = getFolderSize(MYID)
  File "C:\src\html\SEGCloud\index.py", line 13, in getFolderSize
    total_size = os.path.getsize(folder)
  File "C:\Python38\lib\genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [WinError 2] The system cannot find the file specified: '0'
[01/Oct/2021:05:59:08] HTTP 
Request Headers:
  Remote-Addr: 127.0.0.1
  HOST: zuuvs3xsp57e42a5c5lqoby5sli5hxfvbbezfgnfsqopjwqlribyggad.onion
  USER-AGENT: Mozilla/5.0 (Windows NT 10.0; rv:78.0) Gecko/20100101 Firefox/78.0
  ACCEPT: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
  ACCEPT-LANGUAGE: en-US,en;q=0.5
  ACCEPT-ENCODING: gzip, deflate
  CONNECTION: keep-alive
  UPGRADE-INSECURE-REQUESTS: 1
[01/Oct/2021:19:58:21] ENGINE Bus STARTING
[01/Oct/2021:19:58:21] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:19:58:22] ENGINE Serving on http://127.1.1.1:1000
[01/Oct/2021:19:58:22] ENGINE Bus STARTED
[01/Oct/2021:20:07:44] ENGINE Bus STARTING
[01/Oct/2021:20:07:44] ENGINE Started monitor thread 'Autoreloader'.
[01/Oct/2021:20:07:44] ENGINE Serving on http://127.1.1.1:1000
[01/Oct/2021:20:07:44] ENGINE Bus STARTED
[02/Oct/2021:09:48:23] ENGINE Bus STARTING
[02/Oct/2021:09:48:23] ENGINE Started monitor thread 'Autoreloader'.
[02/Oct/2021:09:48:24] ENGINE Serving on http://127.1.1.1:1000
[02/Oct/2021:09:48:24] ENGINE Bus STARTED
[02/Oct/2021:13:14:42] ENGINE Bus STARTING
[02/Oct/2021:13:14:42] ENGINE Started monitor thread 'Autoreloader'.
[02/Oct/2021:13:14:42] ENGINE Serving on http://127.1.1.1:1000
[02/Oct/2021:13:14:42] ENGINE Bus STARTED
[03/Oct/2021:07:06:09] ENGINE Bus STARTING
[03/Oct/2021:07:06:09] ENGINE Started monitor thread 'Autoreloader'.
[03/Oct/2021:07:06:10] ENGINE Serving on http://127.1.1.1:1000
[03/Oct/2021:07:06:10] ENGINE Bus STARTED
[03/Oct/2021:07:07:38] HTTP 
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "C:\Python38\lib\site-packages\cherrypy\_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
  File "C:\Python38\lib\site-packages\cherrypy\lib\encoding.py", line 219, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "C:\Python38\lib\site-packages\cherrypy\_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "index.py", line 97, in home
    byt = getFolderSize(MYID)
  File "index.py", line 13, in getFolderSize
    total_size = os.path.getsize(folder)
  File "C:\Python38\lib\genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [WinError 2] The system cannot find the file specified: '0'
[03/Oct/2021:07:07:38] HTTP 
Request Headers:
  Remote-Addr: 127.0.0.1
  HOST: zuuvs3xsp57e42a5c5lqoby5sli5hxfvbbezfgnfsqopjwqlribyggad.onion
  USER-AGENT: Mozilla/5.0 (Windows NT 10.0; rv:78.0) Gecko/20100101 Firefox/78.0
  ACCEPT: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
  ACCEPT-LANGUAGE: en-US,en;q=0.5
  ACCEPT-ENCODING: gzip, deflate
  CONNECTION: keep-alive
  UPGRADE-INSECURE-REQUESTS: 1
[03/Oct/2021:16:49:52] ENGINE Bus STARTING
[03/Oct/2021:16:49:52] ENGINE Started monitor thread 'Autoreloader'.
[03/Oct/2021:16:49:53] ENGINE Serving on http://127.1.1.1:1000
[03/Oct/2021:16:49:53] ENGINE Bus STARTED
